docs/controller-skills/military.md

```diff
@@ -69,10 +71,33 @@ These Restricted Areas may be activated in vatSys, and the controllers of these
 Unless otherwise stated in Local Instructions, Restricted Areas activated for control by Military TCUs and Towers, will become **Class C** airspace.
 
 ## Formation Flights
-Reserved.
+A standard military formation consits of a proximity of no more than 1 mile laterally or longitudinally and within 100 feet vertically from the flight leader is maintained by each wingman. As such you treat both aircraft as one and ensure only the leader is on mode C. Futhermore per FIHA, Military AIrcraft are given discrete sqwaks in the 5000's and 6000's.
+
+!!! example
+    **VIPR11**: "Viper Flight Checking in"
```

Comment:
The callsign of a formation flight would just be "VIPR" instead of "VIPR11". and the Radio Telephony would be just "Viper" instead of "Viper Flight"